### PR TITLE
Feature/albumin

### PR DESCRIPTION
**Pull request by Murmelmannen**
# Guidance

Another developer and a domain expert (doctor) should review and approve the Pull Request before merge.

### Checklist

- [x] I have added unit tests for happy paths
- [x] I have added unit tests for sad paths
- [x] I have updated the README
- [x] I have added relevant references to external sources in the PR description (such as authors and external links)

Källor: 1) Karolinksa Universitetssjukhuset.
- https://www.karolinska.se/pta/klinisk-kemi/calcium-fosfatprodukt-p-/
2) Makvandi, 2021.
https://anvisningar.se/Anvisningar/Klinisk-kemi/C/P--Calcium/

### CODE REVIEW

File: src/albumin.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        throw new Error('Invalid millimole for corrected calcium')
+    }
+    if (!isValidGram(gram)) {
+        throw new Error('Invalid gram for corrected calcium')
+    }
+    
+    return Math.round(millimole + 0.02 * (40 - gram))
+}
+
+export {correctedCalcium, isValidMillimole}
```
Comment (martify) [inline src/albumin.ts:19]:
```suggestion
export {correctedCalcium, isValidMillimole, isValidGram}
```

File: src/albumin.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+function correctedCalcium(millimole: number, gram: number) {
+    if (!isValidMillimole(millimole)) {
+        throw new Error('Invalid millimole for corrected calcium')
+    }
+    if (!isValidGram(gram)) {
+        throw new Error('Invalid gram for corrected calcium')
+    }
+    
+    return Math.round(millimole + 0.02 * (40 - gram))
```
Comment (martify) [inline src/albumin.ts:16]: How is the rounding? 

Will input value 2 and 3 return 2.7 or 3? 

Old calculator returns 2.7

**Review by martify — APPROVED** [review]
One comment about rounding.

File: src/albumin.ts (hunk 1 above)
Comment (Murmelmannen) [reply src/albumin.ts:19]: Thanks, will add that

File: src/albumin.ts (hunk 2 above)
Comment (Murmelmannen) [reply src/albumin.ts:16]: Yes it seems I missed it. It would be good to add that

**Pull request merged**